File: fizzbuzz/C/fizzbuzz.c
// Compile with cc fizzbuzz.c and run with ./a.out
// Classic fizzbuzz problem from 1 to 100

#include <stdio.h>

void main(){

    for(int i = 1; i <= 100; i++){
        if(i % 3 == 0 && i % 5 == 0){
            printf("FizzBuzz\n");
        }
        else if(i % 3 == 0){
            printf("Fizz\n");
        }
        else if(i % 5 == 0){
            printf("Buzz\n");
        }
        else{
            printf("%d\n", i);
        }
    }

}

File: fizzbuzz/Scala/fizzbuzz.scala
// Compile with scalac fizzbuzz.scala and run with scala fizzbuzz (omit .class)
// Classic fizzbuzz problem from 1 to 100

object fizzbuzz extends App {
    for(i <- 1 to 100) {
        if(i % 3 == 0 && i % 5 == 0) {
            println("FizzBuzz");
        } else if (i % 3 == 0) {
            println("Fizz");
        } else if (i % 5 == 0) {
            println("Buzz");
        } else {
            println(i);
        }
    }
}
